ui/src/views/iam/AddUser.vue

```diff
@@ -222,7 +222,19 @@ export default {
   },
   created () {
     this.form = this.$form.createForm(this)
+<<<<<<< HEAD
     this.apiParams = this.$getApiParams('createUser', 'authorizeSamlSso')
+=======
+    this.apiConfig = this.$store.getters.apis.createUser || {}
+    this.apiParams = {}
+    this.apiConfig.params.forEach(param => {
+      this.apiParams[param.name] = param
+    })
+    this.apiConfig = this.$store.getters.apis.authorizeSamlSso || { params: [] }
+    this.apiConfig.params.forEach(param => {
+      this.apiParams[param.name] = param
+    })
+>>>>>>> 4.15
```

Comment:
diff in a commit can't be good

Reply:
@DaanHoogland isn't that fixed already in the last commit?

Reply:
i see now, it is outdated. Don't know what happened but fine ;)